temporian/implementation/numpy/data/display_utils.py

```diff
@@ -0,0 +1,199 @@
+
+    return (
+        f"indexes: {evset.schema.indexes}\n"
+        f"features: {evset.schema.features}\n"
+        "events:\n"
+        f"{data_repr}\n"
+        f"memory usage: {string.pretty_num_bytes(evset.memory_usage())}\n"
+    )
+
+
+def display_html(evset: EventSet) -> str:
+    """HTML representation, mainly for IPython notebooks."""
+    from xml.dom import minidom
+
+    def create_table_row(
+        columns: List[str], header: bool = False
+    ) -> minidom.Element:
+        tr = dom.createElement("tr")
+        for col in columns:
+            if header:
+                td = dom.createElement("th")
+                text = dom.createElement("b")
+                text.appendChild(dom.createTextNode(col))
+            else:
+                td = dom.createElement("td")
+                text = dom.createTextNode(col)
+            td.appendChild(text)
+            tr.appendChild(td)
+        return tr
+
+    # Create DOM
+    impl = minidom.getDOMImplementation()
+    assert impl is not None
+    dom = impl.createDocument(None, "div", None)
+    top = dom.documentElement
+
+    # Set 0 -> None to disable slice limits (show all)
+    max_indexes = config.max_display_indexes or None
+    max_events = config.max_display_events or None
+    max_features = config.max_display_features or None
+
+    # Other configs
+    convert_datetime = evset.schema.is_unix_timestamp
+    visible_features = evset.schema.features[:max_features]
+    index_schemas = evset.schema.indexes
+    all_index_keys = evset.get_index_keys(sort=True)
+    total_indexes = len(all_index_keys)
+    total_features = len(evset.schema.features)
+    hiding_feats = max_features is not None and total_features > max_features
+
+    # Create one table and header per index value
+    for index_key in all_index_keys[:max_indexes]:
+        # Index header text (n events x n features)
+        index_text = ", ".join(
+            [
+                f"{idx.name}={repr_value(val, idx.dtype)}"
+                for val, idx in zip(index_key, index_schemas)
+            ]
+        )
+        index_data = evset.data[index_key]
+        index_n_events = len(index_data.timestamps)
+        title = dom.createElement("h3")
+        title.appendChild(dom.createTextNode(f"Index: ({index_text})"))
+        descript = dom.createTextNode(
+            f"{index_n_events} events × {total_features} features"
+        )
+        top.appendChild(title)
+        top.appendChild(descript)
+
+        # Table with column names
+        table = dom.createElement("table")
+        col_names = ["timestamp"]
+        col_names += [feature.name for feature in visible_features]
+        if hiding_feats:
+            col_names += ["..."]
+        table.appendChild(create_table_row(col_names, header=True))
+
+        # Rows with events
+        for i, timestamp in enumerate(index_data.timestamps[:max_events]):
+            row = []
+
+            # Timestamp column
+            timestamp = (
```

Comment:
Redefining timestamp with another type make reading the code harder.
What about "raw_timestamp" or "timestamp_repr" (or something like that)?